spec.bs

```diff
@@ -2139,7 +2163,18 @@ Issue: This will require a RFC to add a test-only function to the WPT web driver
   1. Return <b>allowed</b>.
 </div>
 
-<h3 id=disable-fetch>Fetch monkeypatches for network revocation</h3>
+<h3 id=disable-monkeypatches>Monkeypatches for network revocation</h3>
+
+*This introductory section is non-normative.*
+
+The network revocation mechanism requires various patches to various standards. These standards
```

Comment:
How about this instead, if you agree with it:

The network revocation mechanism in this section requires patching standards that make network requests. Before our patches, these standards make network requests (such as initiate WebSocket connections) while unaware of the network revocation *status* of the context that ultimately initiated the request. Our patches add checks to the algorithm in those standards to consider the network revocation status of the initiating context before the request is made.

WDYT? I think it's more clear and less hand wavey.

Reply:
That sounds good. I've updated this section to have that wording.